libminifi/include/utils/GeneralUtils.h

```diff
@@ -43,6 +43,7 @@ using std::make_unique;
 
 template<typename T, typename = typename std::enable_if<std::is_integral<T>::value>::type>
 T intdiv_ceil(T numerator, T denominator) {
+  gsl_Expects(denominator != 0);
   // note: division and remainder is 1 instruction on x86
   return numerator / denominator + (numerator % denominator > 0);
```

Comment:
I know this is old code, but unless we restrict the domain of this function to non-negative arguments, it might give incorrect results, e.g. `intdiv_ceil(-5, -3) == 1`